internal/hash/hash.go

```diff
@@ -0,0 +1,23 @@
+package hash
+
+import (
+	"fmt"
+
+	"github.com/go-task/task/v3/taskfile"
+	"github.com/mitchellh/hashstructure"
```

Comment:
Import grouping should be ordered by: stdlib, libs, Task packages.

```suggestion
	"github.com/mitchellh/hashstructure"

	"github.com/go-task/task/v3/taskfile"
```